rtm/travel_time.py

```diff
@@ -352,6 +352,17 @@ def celerity_travel_time(grid, st, celerity=343, dem=None):
     # Pre-define this boolean for speed
     projected = grid.UTM
 
+    # Clamp station elevations to the DEM surface
+    for tr in st:
```

Comment:
Just realized that there needs to be a check to see if `dem` is not `None` here. Otherwise code will fail.

Reply:
> Okay, I see. I'm inclined to write a small function `clamp_station_to_dem(st, dem)` in `grid.py` and then import that here. That way the tool can be stand-alone since I think it'd be useful. Requirement is that `tr.stats.utm_x` and `tr.stats.utm_y` are defined. Okay if I drive for a bit?

Sure, that seems reasonable. Good catch on the dem is not None either.